### ISSUE TITLE
AWS Feature Release: NodeJS 10.x

### ISSUE THREAD

**Issue by Clete2** [terraform-providers/terraform-provider-aws#8644]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

AWS [recently released support for the NodeJS 10.x Lambda execution environment](https://aws.amazon.com/about-aws/whats-new/2019/05/aws_lambda_adds_support_for_node_js_v10/).

Allow `aws_lambda_function` resource to use `nodejs10.x` runtime.

### New or Affected Resource(s)

* aws_lambda_function

### Potential Terraform Configuration

```hcl
resource "aws_lambda_function" "my_awesome_lambda" {
    ...
    runtime = "nodejs10.x"
}
```

### References

https://aws.amazon.com/about-aws/whats-new/2019/05/aws_lambda_adds_support_for_node_js_v10/

**Comment by Clete2** [comment]
Just found a commit which added this: https://github.com/terraform-providers/terraform-provider-aws/commit/6d39f615e685fc79216d53421e94d592c872cf1b

Wondering when the next release will be cut to include the change?

**Comment by bflad** [comment]
Hi @Clete2 👋 Thank you for this feature request. As you found, this was completed as part of #8627 and #8622. It will be released in version 2.11.0 of the Terraform AWS Provider, likely tomorrow.

**Comment by Clete2** [comment]
Thanks @bflad !

**Comment by willchertoff** [comment]
+1 Thanks @bflad

**Comment by bflad** [comment]
This has been released in [version 2.11.0 of the Terraform AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#2110-may-17-2019). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
